release-please-config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+  "packages": {
+    "packages/aa": {},
+    "packages/allow-scripts": {},
+    "packages/browserify": {},
+    "packages/core": {},
+    "packages/lavapack": {},
+    "packages/node": {},
+    "packages/preinstall-always-fail": {},
+    "packages/tofu": {},
+    "packages/viz": {}
+  },
+  "separate-pull-requests": true
```

Comment:
question related to this above in: https://github.com/LavaMoat/LavaMoat/pull/630/files#r1277501623

Reply:
see https://github.com/LavaMoat/LavaMoat/pull/630#discussion_r1277921360